builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,179 @@
+
+This guide will show you how to access the API endpoints for Moonbeam using curl commands and the Moralis SDK using JavaScript and Python snippets.
+
+--8<-- 'text/_disclaimers/third-party-content-intro.md'
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To interact with Moralis' API endpoints, an account and API key are required.
+
+Head to the [sign up page](https://admin.moralis.io/register){target=\_blank} to create an account. Once you're in, navigate to the **API keys** section. There, you can generate your own unique API key. Be sure to copy it for future use.
+
+![Moralis API key](/images/builders/integrations/indexers/moralis/moralis-1.webp)
+
+## Querying the EVM API {: #querying-the-evm-api }
+
+With the API key, you can try Moralis' REST APIs. The following examples show how the EVM API works:
+
+Get the token balance of a wallet:  
+
+```bash
+curl --request GET \
+--url 'https://deep-index.moralis.io/api/v2.2/0xd4d7fb1f98dD66f6D1f393E8e237AdF74c31F3ea/erc20?chain=moonbeam' \
+--header 'accept: application/json' \
+--header 'X-API-Key: INSERT_YOUR_API_KEY' 
+```
+
+Get the list of owners of an NFT:  
+
+```bash
+curl --request GET \
+--url 'https://deep-index.moralis.io/api/v2.2/nft/0xfffffffffffffffffffffffffffffffffffffffff/owners?chain=moonbeam&format=decimal' \
+--header 'accept: application/json' \
+--header 'X-API-Key: INSERT_YOUR_API_KEY' 
+```
+
+For a comprehensive overview of EVM APIs and their capabilities, please refer to Moralis' [official documentation](https://docs.moralis.io/web3-data-api/evm/reference){target=\_blank}.
+
+## Using the Moralis SDK {: #using-the-moralis-SDK }
+
+Moralis has an SDK that allows developers to seamlessly integrate Moralis' API into their backend infrastructure. This SDK offers a wide array of features, including:
+
+- Data querying from EVM APIs  
+- Integration of Web3 authentication  
+- A collection of utility functions for efficient data transformation and formatting  
+
+Moralis currently provides official SDK support for two primary programming languages:
+
+- NodeJS
+- Python
+
+To install Moralis SDK:
+
+=== "npm"
+
+    ```bash
+    npm install moralis
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add moralis
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm add moralis
+    ```
+
+=== "pip"
+
+    ```bash
+    pip install moralis
+    ```
+
+Once the SDK is ready, you can leverage it to query Moralis APIs. Here's an example of how to interact with the EVM API using a JSON-RPC interface:
+
+=== "JavaScript"
+
+    ```js
+    import Moralis from 'moralis';
+
+    try {
+    await Moralis.start({
+      apiKey: 'INSERT_YOUR_API_KEY',
+    });
+
+    const response = await Moralis.EvmApi.block.getBlock({
+      chain: '0x504',
+      blockNumberOrHash: '1',
+    });
+
+    console.log(response.raw);
+    } catch (e) {
+    console.error(e);
+    }
```

Comment:
This is still not formatted correctly. Are you using prettier? We can sync to get your vs code set up properly, just let me know if that would help! 🙂

Reply:
Is this the correct version? I reinstalled prettier. 
```suggestion
    try {
        await Moralis.start({
            apiKey: 'INSERT_YOUR_API_KEY',
        });
    
        const response = await Moralis.EvmApi.block.getBlock({
            chain: '0x504',
            blockNumberOrHash: '1',
        });
    
        console.log(response.raw);
    } catch (e) {
        console.error(e);
    }
```